cmake/modules/SwiftCompilerCapability.cmake

```diff
@@ -0,0 +1,51 @@
+# Test if the Swift compiler returns success for supplied compiler arguments....
+function(swift_supports_compiler_arguments out_var)
+  file(WRITE "${CMAKE_BINARY_DIR}/tmp/dummy.swift" "")
+  execute_process(
+    COMMAND "${CMAKE_Swift_COMPILER}" -parse ${ARGN} -
+    INPUT_FILE "${CMAKE_BINARY_DIR}/tmp/dummy.swift"
+    OUTPUT_QUIET ERROR_QUIET
+    RESULT_VARIABLE result
+  )
+  if(NOT result)
+    set("${out_var}" "TRUE" PARENT_SCOPE)
+  else()
+    set("${out_var}" "FALSE" PARENT_SCOPE)
+  endif()
+endfunction()
+
+# Test if the Swift compiler supports -disable-implicit-<module>-module-import.
+macro(swift_supports_implicit_module module_name out_var)
```

Comment:
(Optional) There is a function with the same name in `cmake/modules/SwiftImplicitImport.cmake` -- we may consider folding that into this new implementation.

Reply:
This PR removes `cmake/modules/SwiftImplicitImport.cmake` and replaces `swift_supports_implicit_module` function with this new implementation.